CAP/gap/CAP.gd

```diff
@@ -335,12 +335,15 @@ DeclareGlobalFunction( "DeactivateCachingOfCategory" );
 
 #! @BeginGroup
 #! @Description
-#!  Most basic operations have a prefunction, which does a (sometimes partial)
-#!  typecheck at the beginning of the operation. These functions enable or disable this check
-#!  for a category. (Enabled by default)
+#!  Most operations have prefunctions, that perform additional checks on
+#!  the arguments. The checks can either be partial (enabled by default),
+#!  full, or disabled. With the following commands you can either
+#!  enable the full checks, just the basic checks, or, for performance,
+#!  disable the checks alltogether.
 #! @Arguments category
 DeclareGlobalFunction( "DisableBasicOperationTypeCheck" );
 DeclareGlobalFunction( "EnableBasicOperationTypeCheck" );
+DeclareGlobalFunction( "FullBasicOperationTypeCheck" );
```

Comment:
What about the following names for these functions?
`DisableBasicOperationTypeCheck`
`EnablePartialBasicOperationTypeCheck`
`FullBasicOperationTypeCheck`.

To have backwards compatibility,
maybe `EnableBasicOperationTypeCheck`
can be a synonym for `EnablePartialBasicOperationTypeCheck`.

Reply:
OK